### PR TITLE
81 global settings

### PR DESCRIPTION
**Pull request by katypool**
issue #81

### CODE REVIEW

**Comment by pirog** [comment]
This all looks very good. nice call on entity reference in form api! 

a couple of small things which i will post inline in a sec

File: sites/all/modules/playbox_admin/playbox_admin.module [hunk 1]
```diff
@@ -0,0 +1,105 @@
+<?php
+/**
+ * @file
+ * Code for the Playbox Global Settings module.
+ */
+
+function playbox_admin_menu()	 {
```
Comment (pirog) [inline sites/all/modules/playbox_admin/playbox_admin.module]: You are going to want to get your sublime text editor to match up with drupal coding standards. For example tabs = spaces and indents are 2 tabs. You can find good docs on settings that up here:
https://kalamuna.atlassian.net/wiki/display/KALA/Sublime+Text+2+configuration

You also are going to want to make sure nicely closing arrays and things of that sort for example:

```php
function playbox_admin_menu() {
  $items['admin/config/playbox'] = array(
    'title' => 'Playbox',
    'position' => 'left',
    'weight' => 200,
    'page callback' => 'system_admin_menu_block_page',
    'access arguments' => array('administer site configuration'),
    'file' => 'system.admin.inc',
    'file path' => drupal_get_path('module', 'system'),
  );
  return $items;
}
```

File: sites/all/modules/playbox_admin/playbox_admin.install.php [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?php
```
Comment (pirog) [inline sites/all/modules/playbox_admin/playbox_admin.install.php]: this file needs to be called `playbox_admin.install` and it needs a `hook_uninstall()` that removes all the `playbox_admin` variables when you uninstall the module.

**Comment by pirog** [comment]
@katypool let's clean up the above here before we do anything else. then we can merge all this fun stuff in. close the related issues and move onto ye old ONE BRANCH/ONE PULL REQUEST/ONE ISSUE model 

sound good?

**Comment by katypool** [comment]
@pirog added some PIZZAZZ to my Sublime, so now everything will be beautiful from now on (I hope). Still not %100 sure I got the hook_uninstall correct. Do I need to tell it what table to find the variables in?

**Comment by pirog** [comment]
@katypool styling looks better here but lets make sure we are actually using the normal drupal standards and getting these files up to date before we merge.

can help you set this up come Monday!

**Pull request merged**